.github/workflows/linux.yml

```diff
@@ -31,6 +31,16 @@ jobs:
           sudo apt install -y \
             doxygen libxcb-xkb-dev valgrind ninja-build \
             libwayland-dev wayland-protocols bison graphviz
+      - name: Install xkeyboard-config
+        run: |
+          # HACK: Force xkeyboard-config version to master
```

Comment:
It is not clear if the hack is the fact that xkeyboard-config is forced to master, or how it's done. I understand it's the latter, but let's clarify.

Reply:
Fixed